pkg/console/subresource/configmap/configmap.go

```diff
@@ -193,3 +193,13 @@ func authServerYaml(logoutRedirect string) yaml.MapSlice {
 func consoleBaseAddr(host string) string {
 	return util.HTTPS(host)
 }
+
+// Helper function that pulls the yaml struct out of the data section of a configmap yaml
+func getDataFromConfigMapAsYaml(managedConfig *corev1.ConfigMap) []byte {
```

Comment:
Could just call this `extractYAML` if you want to go shorter but no big deal.  Probs should caps `YAML` either way.